### ISSUE TITLE
New user empty screen

### ISSUE THREAD

**Issue by amulware** [toggl/mobileapp#3733]
This screen is shown to users who just _signed up_.

## Design
![](https://user-images.githubusercontent.com/5064445/42080537-b1b3b7f6-7b8b-11e8-8cf5-b911fff7750a.png)

Zeplin: https://zpl.io/bAGqq6n

**Comment by daividssilverio** [comment]
Only for new users? I'd argue that this doesn't look like an empty screen, but a loading placeholder instead. Won't users expect something to appear after seeing this? This is a common pattern for loading placeholders.

**Comment by juanlaube** [comment]
I agree, it's a common loading pattern (and personally, I don't like fake loading screens). But this is also consistent with iOS.

**Comment by lizaPlotnikova** [comment]
@daividssilverio 
While I do think that you make a good point about this being a common pattern for loading states, I'm not sure I share your concern with it being confusing to new users.
• The new user empty state design on Android is very similar to the one on iOS.
    Is the iOS version already implemented? If so, is there any evidence that the state is causing friction for new users? – (we can compare the time to click on "Start" between the two platforms for example)
    If we were to re-think these, should we think about changing it on both platforms, or should we have some kind of comparative KPS for the two versions?
• Unless we go with something out-of-the box (like google's[ quickstart model, button highlight](https://material.io/design/communication/onboarding.html#quickstart-model) ) we should probably take some time to come up with new designs and review those. I'm otherwise engaged at the moment (re-working the Suggestions designs) please, take a number ;P

**Comment by juanlaube** [comment]
@lizaPlotnikova the iOS version was implemented long time ago. I think we should implement this as it is and review later for both apps.

**Comment by daividssilverio** [comment]
You guys are right @lizaPlotnikova @juanlaube . Let's implement it and see what happens. We can do some tracking later, I don't think I've ever seen any complaints from support either.

By the way, the toast to offer education looks like a nice idea to me (but that's another story).
